src/components/text.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+import styled from "styled-components"
+import * as fonts from "../assets/fonts"
+
+type TextAligns = "start" | "center" | "end"
+type TextSizes = "small" | "large"
+
+interface TextProps extends React.HTMLProps<JSX.Element> {
```

Comment:
If you’re using VSCode, you can command+click one of these element types and you’ll be taken to the definitions file where you will find all the others too.